src/transport/mod.rs

```diff
@@ -19,11 +20,12 @@ pub use self::{
 
 fn set_nodelay_or_warn(socket: &::tokio::net::TcpStream) {
```

Comment:
Nit: now that we've added an import for `tokio::net::TcpStream`, why not just change this to
```suggestion
fn set_nodelay_or_warn(socket: &TcpStream) {
```